src/vs/workbench/api/common/extHostTypeConverters.ts

```diff
@@ -2843,6 +2843,8 @@ export namespace ChatResponsePart {
 			return ChatResponseCodeCitationPart.from(part);
 		} else if (part instanceof types.ChatResponseMovePart) {
 			return ChatResponseMovePart.from(part);
+		} else if (part instanceof types.ChatResponseNotebookEditPart) {
+			// TODO@DonJayamanne
```

Comment:
Something is wrong here, it seems like the extension pushes this type but it's never converted to an internal type, please take a look @DonJayamanne

Reply:
Will take a look

Reply:
See Line 2832